modules/pipeline/pipengine/reconciler/queuemanage/types/queue.go

```diff
@@ -29,4 +29,5 @@ type Queue interface {
 	RangePendingQueue()
 	AddPipelineIntoQueue(p *spec.Pipeline, doneCh chan struct{})
 	PopOutPipeline(p *spec.Pipeline)
+	UpdatePipelinePriority(p *spec.Pipeline, priority int64)
```

Comment:
- use mgr.`AddPipelineIntoQueue`
- `defaultQueue` add flag `pendingRangingQueue` when batch upgrade priorities, use `defer` to unset
- test with a large time sleep